.github/workflows/build-wasm.yml

```diff
@@ -0,0 +1,38 @@
+name: WASM build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  build:
+    name: "WebAssembly build ${{ github.event.ref }} for commit ${{ github.event.inputs.commit }}"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Output link to real commit
+        run: echo ${{ github.repository }}/commit/${{ github.event.inputs.commit }}
+
+      - name: Checkout ${{ github.event.inputs.commit }}
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.commit }}
```

Comment:
same here

Reply:
Try taking it out and see if it still works.

Reply:
I took it out and the code isn't being checked out, so let's keep it in^

Reply:
Only using the `uses: actions/checkout@v2` line as a job here should be fine - you only need explicit input usage when you are doing a `workflow_dispatch` or something and not a `push` or `pull_request` event.